docs/pai-management/doc/how-to-setup-dev-box.md

```diff
@@ -70,7 +87,6 @@ sudo docker build -t dev-box . --file=dev-box.dockerfile
 sudo docker run -itd \
```

Comment:
make sure each parameter is needed, simpler is better.

Reply:
Yes, all the parameter is necessary.